packages/styles/src/components/form-footer.scss

```
@@ -0,0 +1,36 @@
+@forward './../variables/options';
+@use './../themes/bootstrap/core' as *;
+
+@use '../variables/color';
+@use '../variables/commons';
+@use '../variables/spacing';
+@use './../variables/breakpoints';
+@use '../mixins/utilities' as utility-mx;
+@use '../tokens/temp-form-footer' as components;
+@use '../functions/tokens' as tokens;
+
+tokens.$default-map: components.$post-form-footer;
+
+.form-footer {
+  border-top: tokens.get('post-form-footer', 'post-form-footer-border-top-width') solid
+    tokens.get('post-form-footer', 'post-form-footer-border-top-color');
+  padding-top: tokens.get('post-form-footer', 'post-form-footer-padding-top-s');
```

We recently decided to move to inline/block properties instead of top/bottom/right/left.
```suggestion
  padding-block-start: tokens.get('post-form-footer', 'post-form-footer-padding-top-s');
```